docs/en/observability/apm/shared-ssl-logstash-config.asciidoc

```diff
@@ -73,43 +73,43 @@ Before running APM Server, you should validate the {ls} server's certificate. Yo
 
 [source,shell]
 ------------------------------------------------------------------------------
-curl -v --cacert ca.crt https://logs.mycompany.com:5044
+curl -v --cacert ca.crt <YOUR_URL>:5044
 ------------------------------------------------------------------------------
 
 If the test is successful, you'll receive an empty response error:
 
 [source,shell]
 ------------------------------------------------------------------------------
-* Rebuilt URL to: https://logs.mycompany.com:5044/
+* Rebuilt URL to: <YOUR_URL>:5044/
 *   Trying 192.168.99.100...
-* Connected to logs.mycompany.com (192.168.99.100) port 5044 (#0)
+* Connected to <YOUR_URL> (192.168.99.100) port 5044 (#0)
 * TLS 1.2 connection using TLS_DHE_RSA_WITH_AES_256_CBC_SHA
-* Server certificate: logs.mycompany.com
-* Server certificate: mycompany.com
+* Server certificate: <YOUR_URL>
+* Server certificate: <YOUR_URL>
 > GET / HTTP/1.1
-> Host: logs.mycompany.com:5044
+> Host: <YOUR_URL>:5044
 > User-Agent: curl/7.43.0
 > Accept: */*
 >
 * Empty reply from server
-* Connection #0 to host logs.mycompany.com left intact
+* Connection #0 to host <YOUR_URL> left intact
 curl: (52) Empty reply from server
 ------------------------------------------------------------------------------
 
 The following example uses the IP address rather than the hostname to validate the certificate:
 
 [source,shell]
 ------------------------------------------------------------------------------
-curl -v --cacert ca.crt https://192.168.99.100:5044
+curl -v --cacert ca.crt <YOUR_IP_ADDRESS>:5044
```

Comment:
For IP addresses we have https://github.com/elastic/docs-content-internal/issues/152 that will get us covered. No need to use placeholders for these examples, we can instead use IP addresses designed for docs examples